libtcmu.c

```diff
@@ -277,78 +268,43 @@ static int generic_handle_cmd(struct tcmu_device *dev,
 	case MODE_SELECT:
 	case MODE_SELECT_10:
 		return tcmu_emulate_mode_select(cdb, iovec, iov_cnt, sense);
+	default:
+		tcmu_err("unknown command %x\n", cdb[0]);
+		return TCMU_NOT_HANDLED;
+	}
+}
+
+static bool command_is_store_call(uint8_t cmd)
+{
```

Comment:
> Above do you mean in general you are opposed to these patches and you are saying we do not need the store callout abstraction in the patchset, or just specifically for compare_and_write?

I'm opposed to callout abstraction in general. Why I introduce passthrough mechanism is to elimilate the need of callout abstraction of this kind. There is no necessity for callout abstraction yet.